frontend/bin/assets/XingWebsite-0.0.1.css

```diff
@@ -3922,9 +3918,99 @@ h4 {
   font-size: 100%;
 }
 
-/* line 3, ../../src/styles/states/_root_homepage.sass */
-#root_homepage h1 {
-  border-bottom: 1px black solid;
+@media screen and (min-width: 0px) {
+  /* line 3, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero {
+    width: 100%;
+    height: 90vh;
+    background-color: black;
+    position: relative;
+  }
+  /* line 8, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero #main-logo {
+    width: 75%;
+    margin: 0 auto;
+    padding-top: 20vh;
+  }
+  /* line 12, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero #main-logo svg {
+    width: 100%;
+    height: 100%;
+  }
+  /* line 15, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero h2 {
+    text-align: center;
+    color: white;
+    font-size: 1.5em;
+    margin: 1em;
+  }
+  /* line 20, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero .red-text {
+    color: #e03237;
+  }
+  /* line 22, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero #get-started {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row;
+    flex-flow: row;
+    -webkit-justify-content: center;
+    justify-content: center;
+    -webkit-align-items: center;
+    align-items: center;
+    background-color: white;
+    border-radius: 5px;
+    height: 3em;
+    width: 10em;
+    margin: 0 auto;
+  }
+  /* line 32, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero #get-started p {
+    text-transform: uppercase;
+    font-size: 1em;
+  }
+  /* line 35, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero #crosswalk {
+    position: absolute;
+    bottom: -4px;
+    left: -5%;
+    width: 110%;
+    margin: 0 auto;
+  }
+  /* line 41, ../../src/styles/states/_root_homepage.sass */
+  #root_homepage #hero #crosswalk svg {
+    width: 100%;
+    height: 100%;
+  }
+}
+@media screen and (min-width: 875px) {
```

Comment:
I might lower the min-width here -- I get the "Get Started" covering the crosswalk around 690 pixels up until it switched to desktop view.